app/services/teacher_training_api/import_course.rb

```diff
@@ -105,16 +105,26 @@ def for_2024_routes
       }
     end
 
+    def for_2025_routes
+      {
+        fee_funded_initial_teacher_training: :"provider_led_#{degree_type}",
+        school_direct_salaried: :school_direct_salaried,
+        postgraduate_teacher_apprenticeship: :pg_teaching_apprenticeship,
+        teacher_degree_apprenticeship: :teacher_degree_apprenticeship,
+      }
+    end
+
     def routes
-      if Settings.current_recruitment_cycle_year < 2024
-        before_2024_routes
-      else
-        for_2024_routes
-      end
+      return before_2024_routes if Settings.current_recruitment_cycle_year < 2024
```

Comment:
We can drop this branch, not used since 2 cycle ago.
We only ever have an overlap between two cycles as we transition from old to new.